ci/code_checks.sh

```diff
@@ -23,5 +23,5 @@ if [ "$skipexpensive" != "true" ]; then
   popd
 
   echo "Type checking"
-  pytype ${SRC_FILES[@]}
+  pytype -P src/:. ${SRC_FILES[@]}
```

Comment:
What problem does adding `src/` and `.` to the PYTHONPATH explicitly fix?

Reply:
When I install in editable (developer) mode, i.e. `pip -e .`, `pytype` non-deterministically fails to detect a significant fraction of modules. It works on other commits, so something in this PR breaks its detection in some way, but I've been unable to figure out what.

I'll add a comment to this effect.